apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -5036,6 +5036,9 @@
     },
     "StellarSourceHasMultiSign": {
       "title": "Please disable multisign to make Stellar transactions"
+    },
+    "HederaAddAccountError": {
+      "title": "Your account has not been added. Please try again."
     }
```

Comment:
Perfect :+1: But can you please do the same for Live Mobile?
https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-mobile/src/locales/en/common.json

Reply:
Oh yes definitely, sorry about that haha.